irods/message/__init__.py

```diff
@@ -14,12 +13,22 @@
 from collections import namedtuple
 import os
 import ast
+import sys
 import threading
 from irods.message.message import Message
 from irods.message.property import (BinaryProperty, StringProperty,
                                     IntegerProperty, LongProperty, ArrayProperty,
                                     SubmessageProperty)
 
+# Unfortunately, the previous import line redefines the "property" attribute within the current
```

Comment:
Can we... not redefine 'property' in the previous import?

Or could/would that be a breaking change for others downstream?

Reply:
Possibly. I'm going to try it

Reply:
If you've handled this, go ahead and mark it as resolved.

Reply:
Nice.    And no downstream effects?    aka, nobody else depending on the current naming?

Reply:
They can still depend on it.  I've defined it as the very last thing in the module, rather than at the head, so that definition is what permanently stands in external uses of the irods.message module.  But internally, this doesn't interfere with previously valid definition of `property`.  Python luckily has a mostly serial and one-pass approach to compiling code, which lets us do that kind of thing.

Reply:
oh nice.   carry on.